manifests/base/workflow-controller/workflow-controller-deployment.yaml

```diff
@@ -16,6 +16,9 @@ spec:
         - name: workflow-controller
           image: argoproj/workflow-controller:latest
           securityContext:
```

Comment:
Any reason not to merge with the pod security context?

Reply:
Theoretically, the pod may have other containers which may not function with those attributes enabled. However, I'll be happy to move this to the pod level if you suggest so.

Reply:
Injected side-cars? Istio? I think I'd like it all in one place. I don't really mind where that is.